boards/kernel_layout.ld

```diff
@@ -137,6 +137,19 @@ SECTIONS
         /* End constructor/destructor */
     } > rom
 
+    /* STATIC ELEMENTS FOR TOCK APPLICATIONS */
+    .apps :
+    {
+        /* _sapps symbol used by tock to look for first application */
+        . = ALIGN(4);
+        _sapps = .;
+
+	LONG(0)
+
+        /* Optional .app sections a convenience mechanism to bundle tock
+           kernel and apps into a single image */
+        KEEP (*(.app.*))
+    } > prog
```

Comment:
Did moving this and adding `LONG(0)` fix an issue you were having?

Reply:
I initially removed `.apps :` in order to compile and test `kernel_loop` without `load_process`. However when I put `.apps :` back, I put it at a different place in `kernel_layout.ld`. Hence it showed up as a move. I'll correct it in the next push.

Regarding `LONG(0)`, this is needed for `arm-none-eabi-objcopy --update-section` to work properly. Without `LONG(0)`, `arm-none-eabi-objcopy` gives the following error.

```
arm-none-eabi-objcopy:target/thumbv7em-none-eabi/debug/nucleo_f446re-app[.apps]: Section has no contents
```